src/core/sys/posix/spawn.d

```diff
@@ -0,0 +1,127 @@
+/**
+ * D header file for spawn.h.
+ *
+ * Copyright: Copyright (C) 2018 by The D Language Foundation, All Rights Reserved
+ * Authors:   Petar Kirov
+ * License:   $(LINK2 https://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:    $(LINK2 https://github.com/dlang/druntime/blob/master/src/core/sys/posix/spawn.d, _spawn.d)
+ * Standards: The Open Group Base Specifications Issue 6, IEEE Std 1003.1, 2004 Edition
+ */
+
+public import core.sys.posix.sys.types : mode_t, pid_t;
+public import core.sys.posix.signal : sigset_t;
+public import core.sys.posix.sched : sched_param;
```

Comment:
According to http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/spawn.h.html:
> * The <spawn.h> header shall define the mode_t and pid_t types as described in <sys/types.h>.
> * The <spawn.h> header shall define the sigset_t type as described in <signal.h>.
> * The tag sched_param shall be declared as naming an incomplete structure type, the contents of which are described in the <sched.h> header.